packages/timeline/README.md

```diff
@@ -0,0 +1,167 @@
+# timeline
+
+## What is this
+
+A React component used in articles at [The Reporter Taiwan](https://www.twreporter.org).
+
+Published as an [npm package](https://www.npmjs.com/package/@twreporter/timeline).
+
+## Data Structure
+
+![timeline example](./timeline.jpg)
+
+### sections
+
+A timeline graph is divided into several sections. There are two levels of section for now:
+
+- `group`: one `unit` section or several ones may constitute a `group` section. Which means `unit` is the subsection of `group`
+- `unit`: the lowest level of section
+
+The background blocks are divided according to one of section types.
+
+A section contain one heading element (optional), and may have multiple subsections or sub-elements.
+
+### elements
+
+_Elements_ are the basic items in sections. In principle, one _element_ corresponds to one _row_ of the data sheet.
+
+There are three types of element for now:
+
+- `group-flag`: the heading element of a group section
+- `unit-flag`: the heading element of a unit section
```

Comment:
`unit-flag` or  `record-flag`? 
I saw https://docs.google.com/spreadsheets/d1f76OLdfZe3kyNOKiPthWNJWVGmY3bkm5KtxB4NYp9uU using `record-flag`.

Reply:
`record-flag` is the old naming of the `unit-flag` (but it still be used by Wuhan coronavirus for now, I'll upgrade the data in that spreadsheet after this package is published)